scripts/create_spec_repo/firebase_sdk.textproto

```diff
@@ -0,0 +1,5 @@
+# Config file to update versions of Pods which will be pushed to SpecsStaging
```

Comment:
Can this file be deleted?

Reply:
> It looks like there's a failing CI - https://github.com/firebase/firebase-ios-sdk/pull/6335/checks?check_run_id=1038350404

This is fixed by https://github.com/firebase/firebase-ios-sdk/pull/6355

> Can this file be deleted?

This file is for some closed source sdk. e.g. FirebaseAnalytics. This will happen in a scenario like when in Firebase.podspec, FirebaseAnalytics is updated to a newer version but it is not released yet, then when we push to a testing repo, it will show error like FirebaseAnalytics version not found and so Quickstart tests cannot run since repo is failed to build. This file can update the version to let tests run before a closed source gets released.
However, if we consider that failure is expected then we can remove that file. One of my concern is manual effort might be needed to update that file.